torch/CMakeLists.txt

```diff
@@ -205,30 +205,30 @@ target_compile_definitions(torch PRIVATE _THP_CORE)
 
 # until they can be unified, keep these lists synced with setup.py
 if(MSVC)
-	if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
-		set (MSVC_RUNTIME_LIBRARY_FLAG "/MDd")
-	else()
-		set (MSVC_RUNTIME_LIBRARY_FLAG "/MD")
-	endif()
-	
-	target_compile_options(torch PRIVATE
-	  ${MSVC_RUNTIME_LIBRARY_OPTION}
-	  /Z7
-	  /EHa
-	  /DNOMINMAX
-	  /wd4267
-	  /wd4251
-	  /wd4522
-	  /wd4522
-	  /wd4838
-	  /wd4305
-	  /wd4244
-	  /wd4190
-	  /wd4101
-	  /wd4996
-	  /wd4275
-	  /bigobj
-	  )
+  if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
```

Comment:
Why do you replace tabs with spaces? The remaining part of the code uses tabs. Let's keep it.

Reply:
the remaining code uses spaces

Reply:
I'm quite happy to see tabs go :)